[UNR-2445][MS] Doc changes for server travel. (#1511)
spatialos/UnrealGDK: SpatialGDK/Documentation/unreal-features-support.md

 <tr>
    <td rowspan="3"><a href="https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling"><b>Travel<b></a></td>
    <td>Server Travel</td>
    <td class="supported"></td>

>     <td>Not supported when used along side zoning or off-loading</td>

Reviewer: ```suggestion
    <td>Not supported when used in conjunction with zoning or offloading</td>
```

Reviewer 2: I think the cell type should be "Supported with caveats or workarounds" not supporte

Reviewer 2: Actually maybe this is fine given the whole table is for single-server support only

>     <td>Not supported when used in conjunction with zoning or offloading</td>

Reviewer 3: Personally, I would have all of the travels as `ServerTravel` and `ClientTravel` and `SeamlessTravel` to be consistent, but I don't want to change the whole doc here.

Suggest use consistent terms and info.

```suggestion
    <td>Not supported with multiserver zoning or offloading. Multiserver support is on the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1#card-22461878).</td>
```
